lte/gateway/Makefile

```diff
@@ -59,7 +59,14 @@ OAI_FLAGS = $(S6A_FLAGS) -DEMBEDDED_SGW=True -DENABLE_OPENFLOW=True -DSPGW_ENABL
 endif
 
 # debian stretch build uses older cc not recognizing options needed on ubuntu focal
-COMMON_FLAGS := $(shell grep -q stretch /etc/os-release || echo -DCMAKE_C_FLAGS='"-Wno-error=duplicate-decl-specifier -Wno-error=maybe-uninitialized -Wno-error=address-of-packed-member"')
+
+OS_VERSION_NAME := $(shell grep VERSION_CODENAME /etc/os-release | sed 's/.*=//g')
```

Comment:
@kkahrs 

sorry to bother again.


CentOS/RHEL systems do not have VERSION_CODENAME in `/etc/os-release` file.

Your line will fail.

Correct to `OS_VERSION_NAME := $(shell grep VERSION_CODENAME /etc/os-release | sed 's/.*=//g' || true)`

or something equivalent.

Raphael

Reply:
fixed